src/utils/url.ts

```diff
@@ -69,3 +69,8 @@ export function getPreviousPageUrl(req: Request, basePath: string) {
 
   return relativePreviousPageUrl;
 }
+
+export const isUpdateOrRemoveJourney = (req: Request): boolean => {
```

Comment:
I've added a comment to @markpit 's PR regarding the location and implementation of this new function. I think it needs to move to the validator code where it's called from.

Reply:
Moved to legal entity validation.